### ISSUE TITLE
Investigate Memcache Behavior

### ISSUE THREAD

**Issue by fangeugene** [the-blue-alliance/the-blue-alliance#350]
Something fishy is going on with memcache. It seems like whenever I check the memcache stats, the oldest item age hovers around 3 hours 30 minutes. It makes sense that some items would get expired when the total cache size gets too large based on the memcache eviction policy (https://developers.google.com/appengine/docs/python/memcache/overview#How_Cached_Data_Expires), with "the least recently used values are evicted first," but one would expect that the main landing page's memcache entry wouldn't be anywhere near the least recently used value considering there are  around 4000 entries in memcache at a given time.

TLDR: Something is going on that leads me to believe we're not using memcache to its maximum potential. The oldest item age should almost always be the age of  the entry for the main landing page, yet it's always hovering at around 3 hours 30 minutes.

**Comment by gregmarra** [comment]
We've got dedicated memcache during the season now.
